scalyr_agent/copying_manager/copying_manager.py

```diff
@@ -793,6 +807,33 @@ def run(self):
                 for worker_pool in self.__api_keys_worker_pools.values():
                     worker_pool.start_workers_and_block_until_copying()
 
+                # if the multi-process workers are enabled, then we have to set the worker PID's for the instances of the
+                # 'linux_process_monitor' that may be set to monitor the worker processes.
+                if self.__config.use_multiprocess_copying_workers:
```

Comment:
I think we should add a config option / feature flag so this functionality (gathering metrics for worker process) can be disabled.

And for now it's probably better to default it to false / off.